requirements/app/base.txt

```diff
@@ -14,8 +14,8 @@ psutil <5.9.5
 click <=8.1.3
 python-multipart>=0.0.5, <=0.0.6
 
-fastapi >=0.69.0, <0.98.0
-starlette  # https://fastapi.tiangolo.com/deployment/versions/#about-starlette
+fastapi >=0.92.0, <0.98.0
+starlette >=0.25.0, <=0.28.0  # https://fastapi.tiangolo.com/deployment/versions/#about-starlette
```

Comment:
@Borda this link (https://fastapi.tiangolo.com/deployment/versions/#about-starlette) explicitly recommends that starlette is not pinned

Reply:
note that the upper bound is only for CI stability, see: https://github.com/Lightning-AI/lightning/tree/master/requirements

Reply:
That makes sense! It has implications for users, too, though; because it’s specified in your requirements, it ends up being enforced by package managers of folks that are consuming Lightning

Reply:
this is manages in `setup.py` so when ever you install it even from source you get lifted the upper bounds